httpclient-processor/src/test/java/org/springframework/cloud/stream/module/httpclient/HttpClientProcessorTests.java

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.module.httpclient;
+
+import static org.hamcrest.Matchers.*;
+import static org.junit.Assert.*;
+import static org.springframework.cloud.stream.test.matcher.MessageQueueMatcher.*;
```

Comment:
No asterisk imports.
`static` imports in the end.
https://github.com/spring-projects/spring-framework/wiki/Spring-Framework-Code-Style.
Even if SF guys uses asterisks, we found it a bit ambiguity when we use several teststing frameworks who has the same statics methods.